ev3dev2/motor.py

```diff
@@ -67,13 +67,19 @@
     raise Exception("Unsupported platform '%s'" % platform)
 
 
-class SpeedValue():
+class SpeedValue(object):
     """
     A base class for other unit types. Don't use this directly; instead, see
     :class:`SpeedPercent`, :class:`SpeedRPS`, :class:`SpeedRPM`,
     :class:`SpeedDPS`, and :class:`SpeedDPM`.
     """
-    pass
+
+    def __lt__(self, other):
+        return self.to_native_units() < other.to_native_units()
+
+    def __rmul__(self, other):
+        return self.__mul__(other)
```

Comment:
So we can multiple SpeedValue objects by an int, float or another SpeedValue object of the same type.  I'm not sure how much use multiplying by another object of the same type will be but being able to scale one by multiple by a float or reverse the direction by multiplying by -1 is handy.

Reply:
Just nitpicking here: a product of two speed values is no longer a speed value, it has units of [m^2]/[s^2].

Reply:
I will simplify this and only support multiplying by int and float I can’t think of a good use case for anything more complicated than that.